plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -259,6 +262,7 @@ private List<DeltaLakeSplit> splitsForFile(
                     fileSize,
                     addFileEntry.getModificationTime(),
                     ImmutableList.of(),
+                    SplitWeight.fromProportion(Math.min(Math.max((double) splitSize / maxSplitSize, minimumAssignedSplitWeight), 1.0)),
```

Comment:
`Math.min( ..., 1)` is redundant per my understanding.

nit: static import min & max

Reply:
> `Math.min( ..., 1)` is redundant per my understanding.

ie if we somehow end up creating an oversized split, it's fine to let the engine know about that.